Dockerfile

```diff
@@ -4,6 +4,7 @@ FROM nvidia/cuda:$CUDA_VERSION-devel-ubuntu22.04
 
 ARG TARGETARCH
 ARG VERSION=0.0.0
+ARG GOFLAGS="'-ldflags -w -s'"
```

Comment:
What do the `-w` and `-s` flags do? I was trying to track those down through the go build docs but didn't see anything.